powerhome/playbook
# [PLAY-970] Website - Changelog updates

[PLAY-970](https://nitro.powerhrg.com/runway/backlog_items/PLAY-970)

This PR updates styling for the `Changelog.md` file

New Styles:
![Screenshot 2023-09-13 at 10 25 26 AM](https://github.com/powerhome/playbook/assets/92755007/0fb57e9c-4546-47c4-a0b7-11892696e98f)


Blockquote Example (this is not actually in blockcode, just added it for an example image):
![Screenshot 2023-09-13 at 10 33 33 AM](https://github.com/powerhome/playbook/assets/92755007/82481639-174e-4352-ab88-bb0c6bd9f074)




**How to test?** Steps to confirm the desired behavior:
1. Go to Changelog page and confirm that new styles are correct.

To Test blockquotes:
1. Pull down branch locally
2. Make something in `Changelog.md` blockquote by adding a `> `  at the start if the line.
3. check local env to see if styles look correct.



#### Checklist:
